pkg/resources/service/create_test.go

```diff
@@ -0,0 +1,31 @@
+package service
+
+import (
+	"bytes"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/triggermesh/tm/pkg/client"
+)
+
+func TestDryRunDeployment(t *testing.T) {
+	buffer := new(bytes.Buffer)
+	Output = buffer
+	defer func() { Output = os.Stdout }()
+
+	client.Dry = true
+	clientset, err := client.NewClient(client.ConfigPath(""))
```

Comment:
I just realized that this client with `client.Dry` will still try to connect to a kubernetes cluster. This worked for me in my local environment because obviously I do have `$HOME/.kube/config.json` with valid credentials / clusters in there. This is not going to work in the continuous environment, though. We need to rework the client behavior with `Dry = true` defined @tzununbekov.

Reply:
Alternatively we can stub HTTP requests in tests to allow requests to `test.any` cluster.

Reply:
Fixed in https://github.com/triggermesh/tm/pull/111